support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.jsx

```diff
@@ -177,7 +177,7 @@ function initialisePaymentRequest(props: PropTypes) {
     currency: props.currency.toLowerCase(),
     total: {
       label: 'The Guardian',
-      amount: props.amount,
+      amount: props.amount * 100,
```

Comment:
This bug didn't manifest because the 'update' function was ran whenever any amount or payment request button was clicked anyway.